libcrux-kem/Cargo.toml

```diff
@@ -25,5 +26,4 @@ pre-verification = ["libcrux-ml-kem/pre-verification"]
 
 [dev-dependencies]
 libcrux-kem = { version = "0.0.2-alpha.1", path = "./", features = ["tests"] }
-libcrux = { version = "0.0.2-alpha.1", path = "../", features = ["rand"] }
 hex = { version = "0.4.3", features = ["serde"] }
```

Comment:
There shouldn't be a version here. That prevents it from being released. It should always take the current version.